addons/redis/scripts/redis-sentinel-start-v2.sh

```diff
@@ -42,6 +42,41 @@ build_redis_sentinel_conf() {
   echo "build redis sentinel conf succeeded!"
 }
 
+check_register_sentinel_conf() {
+  if [ -f /data/sentinel/init_done.conf ]; then
+    echo "normal start"
+  else
+    echo "horizontal scaling"
+    touch /data/sentinel/init_done.conf
+    register_sentinel_conf
+  fi
+}
+
+register_sentinel_conf() {
+
+  if [[ -n "$KB_ITS_0_HOSTNAME" ]]; then
+    sentinel_pod_fqdn="$KB_ITS_0_HOSTNAME"
+    output=$(redis-cli -h "$sentinel_pod_fqdn" -p 26379 -a "$SENTINEL_PASSWORD" sentinel masters)
```

Comment:
`sentinel masters` will retrieve multiple sets of Redis primary and replica instances. Please handle such cases where there are multiple sets of Redis primary and replica instances, each potentially having different usernames and passwords. In the auth-user section below, you need to handle each set separately.

Reply:
I have modified all